pkg/utils/uid.go

```diff
@@ -0,0 +1,16 @@
+package utils
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+)
+
+// GenerateUID generates a random string of 16 bytes
+func GenerateUID() (string, error) {
+	randomBytes := make([]byte, 16)
+	if _, err := rand.Read(randomBytes); err != nil {
+		return "", err
+	}
+
+	return base64.StdEncoding.EncodeToString(randomBytes), nil
+}
```

Comment:
Instead of this func just use `"k8s.io/apimachinery/pkg/util/uuid".UUID` in the plugin package